### PR TITLE
Add gnocchixyz-pastamaker

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 1cbf519f143d08251f756c98bcdcf25e61b5aedf

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 38f12b7870957dc6e3023c596151a79ef45af742

File: gnocchixyz_pastamaker/__init__.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+import requests
+import time
+
+LOG = logging.getLogger(__name__)
+
+
+def get_login_password(site_name="github.com", netrc_file="~/.netrc"):
+    """Read a .netrc file and return login/password for LWN."""
+    n = netrc.netrc(os.path.expanduser(netrc_file))
+    return "sileht-tester", n.hosts[site_name][2]
```
Comment (jd) [inline gnocchixyz_pastamaker/__init__.py]: nop

**Comment by sileht-tester** [comment]
I have put the bot here: https://github.com/sileht/pastamaker.git

I have registered two apps pastamaker and pastamaker-test on heruko and github

pastamaker run on gnocchixyz/tools-tester but the plan is to run it on other gnocchixyz repos.
pastamaker-test run on sileht-tester/tools-tester, this is my devel instance, you can play with it.

Both currently run master, only one reviews is needed for now, I put 2 for gnocchixyz, when we start using it.

So with the bot no more "update branch" and "merge". Just put the required approvals and the bot take care of the rest.

**Pull request closed without merging**